rand_core/src/lib.rs

```diff
@@ -307,6 +329,25 @@ pub trait SeedableRng: Sized {
     /// seeds it is preferable to map these to alternative constant value(s),
     /// for example `0xBAD5EEDu32` or `0x0DDB1A5E5BAD5EEDu64` ("odd biases? bad
     /// seed"). This is assuming only a small number of values must be rejected.
+    /// Alternatively, the newtype pattern can be used to make sure only valid
+    /// seeds can be constructed:
```

Comment:
I don't really like this as a pattern. RNG implementations can do as they wish, but we shouldn't advertise it here i.m.o.

It is useful for generic code to have `from_seed` work with all simple slices. The default implementation of `from_rng`uses this, and something like `from_hashable`would use it it.

Also, does it really work? With the `AsMut` implementation would be able to modify the newtype outside `new()`.

Reply:
> Also, does it really work? With the AsMut implementation wouldn't it be possible to modify the newtype outside new()?

Good point! I removed the suggestion to use the newtype pattern to work around invalid seeds.